### PR TITLE
paradrop crates from helicopter or plane cargo

### PR DESCRIPTION
**Pull request by commy2**

### CODE REVIEW

**Comment by PabstMirror** [comment]
this sounds really cool

Are we planning to add actions for pilots/passangers?

**Comment by commy2** [comment]
No idea yet. I guess for the pilot? Or maybe depending on the type of heli / plane.

**Comment by sargken** [comment]
Crew chief

File: addons/cargo/functions/fnc_paradropItem.sqf [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+_item setPosASL _position;
+
+["hideObjectGlobal", [_item, false]] call EFUNC(common,serverEvent);
+
+// open parachute and ir light effect
+[{
+    params ["_item"];
+
+    if (isNull _item || {getPos _item select 2 < 1}) exitWith {};
```
Comment (jokoho48) [inline addons/cargo/functions/fnc_paradropItem.sqf]: if this is true the code below never gets exectuted are you shure you want to do this?

Comment (commy2) [inline addons/cargo/functions/fnc_paradropItem.sqf]: Yes. I don't want to open a parachute if the crate already landed, e.g. when droping from a very low altitude.

**Comment by commy2** [comment]
- [ ] CBA events

**Comment by PabstMirror** [comment]
@commy2 - integrated this into cargo

Interact with vehicle while mounted.
Shows normal cargo menu (with button name changed)
Requires level flight at a reasonable altitude when unloading.

- [ ] Who should have access to this? Anyone except pilot?
- [ ] Do we want an ace setting for this?

**Comment by commy2** [comment]
I'd say pilot and copilot should be able to access it. It would be too exploitable if the cargo soldiers could do it.
No ACE setting. It already is optional. You don't have to use the action.

**Comment by bux578** [comment]
The Taru has a designated "Loadmaster" seat.

**Comment by jonpas** [comment]
Add a config entry for turrets (like fcs)?

**Comment by esteldunedain** [comment]
I'd say pilot + copilot by default + optional config entry to specify other seats that should be able to do it.

File: addons/cargo/stringtable.xml [hunk 2]
```diff
@@ -201,5 +201,13 @@
             <Portuguese>Tamanho do objeto</Portuguese>
             <Russian>Размер объекта</Russian>
         </Key>
+        <Key ID="STR_ACE_Cargo_paradropButton">
+            <English>Paradrop</English>
+            <German>Abspringen</German>
+        </Key>
+        <Key ID="STR_ACE_Cargo_unlevelFlightWarning">
+            <English>Unlevel Flight</English>
+            <German>Ungleiche Flug</German>
```
Comment (bux578) [inline addons/cargo/stringtable.xml]: `Schieflage`

File: addons/cargo/stringtable.xml [hunk 3]
```diff
@@ -201,5 +201,13 @@
             <Portuguese>Tamanho do objeto</Portuguese>
             <Russian>Размер объекта</Russian>
         </Key>
+        <Key ID="STR_ACE_Cargo_paradropButton">
+            <English>Paradrop</English>
```
Comment (bux578) [inline addons/cargo/stringtable.xml]: Airdrop instead of Paradrop?
https://en.wikipedia.org/wiki/Airdrop

File: addons/cargo/stringtable.xml [hunk 4]
```diff
@@ -201,5 +201,13 @@
...
+            <German>Abspringen</German>
```
Comment (bux578) [inline addons/cargo/stringtable.xml]: I believe the military term is `Türlast` (which translates to "door load").

File: addons/cargo/stringtable.xml [hunk 5]
```diff
@@ -202,12 +202,12 @@
...
         <Key ID="STR_ACE_Cargo_paradropButton">
-            <English>Paradrop</English>
-            <German>Abspringen</German>
+            <English>Airdrop</English>
+            <German>Türlast</German>
         </Key>
         <Key ID="STR_ACE_Cargo_unlevelFlightWarning">
             <English>Unlevel Flight</English>
-            <German>Ungleiche Flug</German>
+            <German>Schieflage Flug</German>
```
Comment (bux578) [inline addons/cargo/stringtable.xml]: Sorry, just "Schieflage"

**Pull request merged**